source/claim.ts

```diff
@@ -18,7 +18,13 @@ export interface ClaimSnak {
 	readonly snaktype: string;
 }
 
-export type ClaimSnakValue = ClaimSnakTimeValue | ClaimSnakEntityValue
+export type ClaimSnakValue = ClaimSnakStringValue | ClaimSnakTimeValue | ClaimSnakEntityValue | ClaimSnakQuantityValue
```

Comment:
I added the suggestion from https://github.com/EdJoPaTo/wikibase-types/pull/15#discussion_r783772352

```suggestion
export type ClaimSnakValue =
       ClaimSnakEntityValue |
       ClaimSnakQuantityValue |
       ClaimSnakStringValue |
       ClaimSnakTimeValue
```